docs/demo.mdx

```diff
@@ -0,0 +1,506 @@
+import HomepageHeader from '@site/src/components/Homepage/Header/HomepageHeader';
+
+import '@site/src/css/custom.css';
+import '@site/node_modules/infima/dist/css/default/default.css';
+
+import FlavorizedCodeBlock from '@site/src/components/FlavorizedCodeBlock';
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# H1 heading
+
+## H2 heading
+
+### H3 heading
+
+#### H4 heading
+
+##### H5 heading
+
+###### H6 heading
+
+## Admonitions:
+
+:::caution CAUTION
+
+This is a CAUTION admonition with a custom background color, and a border color.
+
+:::
+
+:::danger DANGER
+
+This is a DANGER admonition with a custom background color, and a border color.
+
+:::
+
+:::info INFO
+
+This is an INFO admonition with a custom background color, and a border color.
+
+:::
+
+:::success TIP
```

Comment:
What's the difference between this and just `:::tip`?

Reply:
Good catch thank you! It's TIP, I fixed it.